modules/op-sample-example-pipeline-tutorial.adoc

```diff
@@ -0,0 +1,49 @@
+[id='op-sample-example-pipeline-tutorial_{context}']
+= Sample example from pipeline tutorial
```

Comment:
Please use this title: Creating a sample `EventListener` resource using a secure HTTPS connection
Modify 'id' and the include in the assembly as well.

Procedural modules should use imperative headings ('ing') Also titles need to be very clear to the end user. They should be able to assess from the tilte if it is worth digging in deeper. Most users skim the docs before deep diving into content they feel would resolve their issues.